src/main/java/core/basesyntax/strategy/impl/BalanceOperationHandler.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.transaction.FruitTransaction;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction fruitTransaction) {
+        String fruit = fruitTransaction.getFruit();
+        int quantity = fruitTransaction.getQuantity();
+        int newQuantity = Storage.FRUITS.getOrDefault(fruit, 0) + quantity;
+        Storage.FRUITS.put(fruit, newQuantity);
```

Comment:
let's add corresponding validations to all of your handlers. For example, you should check if quantity is valid etc